guides/advanced_guides/settings.md

```diff
@@ -66,9 +66,8 @@ Faceted attributes are the attributes used as facets. They **must be declared at
   An array of strings that contains the attributes to use as facets.
 
 ::: warning
-
-Only fields of data type **string** or **array of strings** can be used for faceting.
-
+  Only fields of data type **string** or **array of strings** can be used for faceting.
```

Comment:
Formatting alignment?
```suggestion
  Only fields of data type **`String`** or **array of `Strings`** can be used for faceting.
```

Reply:
I'm not a huge fan. Is this a good practice ?

Reply:
No I was just wondering. You can resolve it.